tests/org.eclipse.passage.loc.report.core.tests/META-INF/MANIFEST.MF

```diff
@@ -10,4 +10,4 @@ Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Require-Bundle: org.junit;bundle-version="4.12.0",
  org.eclipse.passage.loc.report.core;bundle-version="0.1.0",
  org.eclipse.osgi;bundle-version="3.15.100",
- org.eclipse.passage.lic.users
+ org.eclipse.passage.lic.users;bundle-version="0.5.100"
```

Comment:
Does this bundle exist in the previous version?

Reply:
Not shure I've got the idea of question. Currently `org.eclipse.passage.lic.users` has `0.5.100` version.